### PR TITLE
Dashboard admin config

### PR DESCRIPTION
**Pull request by stephen-lt-sl**
Changes the admin authentication to use a configurable JSON config file instead of a hardcoded variable.

### CODE REVIEW

**Comment by susanthomson** [comment]
in server.js the express middleware and oauth route need to be updated to allow more than one admin

**Comment by susanthomson** [comment]
looks good now

**Comment by stephen-lt-sl** [comment]
Rebased to cover the changes to master since the request was opened

File: server/server.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ module.exports = function(port, googleAuthoriser) {
     function getTweetsWithHashtag() {
         var query = {
             q: hashtags.join(" OR "),
-            since_id: sinceIdH
+            since_id: sinceIdH,
```
Comment (Geor23) [inline server/server.js:98]: no need for ',' here

**Comment by susanthomson** [comment]
:ship:

File: server/server.js (hunk 1 above)
Comment (stephen-lt-sl) [reply server/server.js:98]: There's not really a strict need, but it makes it easier to edit - it's not really a big deal, I only wrote that line on my branch before pulling the fix.

**Pull request merged**